lib/travis_connection.rb

```diff
@@ -0,0 +1,40 @@
+class TravisConnection
+  attr_reader :github_token,
```

Comment:
Those getters/setters don't need to be public, you can either discard them and use the instance variable instead or make them private. I prefer to use the instance variable.